File: commen/src/scala/utils/FileUtils.scala
package utils

import java.io.{File, FileInputStream, FileOutputStream}

import scala.io.Source

class FileUtils {

}
object FileUtils{

  def readFile(path:String)={
    val file = new File(path)
    val fileInputStream = new FileInputStream(file)
    val len = fileInputStream.available
    val pdfBytes = new Array[Byte](len)
    fileInputStream.read(pdfBytes)
    fileInputStream.close()
     pdfBytes
  }
  def writeFile(data:Array[Byte],path:String)={
    val file =new File(path)
    val fileOutputStream=new FileOutputStream(file)
    fileOutputStream.write(data)
    fileOutputStream.flush()
    fileOutputStream.close()
  }

  def main(args: Array[String]): Unit = {
      println("1")
  }

}

File: commen/src/java/com/wx/cloudprint/util/FileUtils.java
package com.wx.cloudprint.util;

import java.io.*;

public class FileUtils {

    public static byte [] readFile(String path) throws Exception {
        File file=new File(path);
        return readFile(file);
    }
    public static byte [] readFile(File file) throws Exception {
        FileInputStream fileInputStream = new FileInputStream(file);
        int len = fileInputStream.available();
        byte [] pdfBytes = new byte[(len)];
        fileInputStream.read(pdfBytes);
        fileInputStream.close();
        return pdfBytes;
    }

    public static void writeFile(byte []data,File file) throws IOException {
        FileOutputStream fileOutputStream=new FileOutputStream(file);
        fileOutputStream.write(data);
        fileOutputStream.flush();
        fileOutputStream.close();
    }
    public static void writeFile(byte []data,String path) throws IOException {
        File file =new File(path);
        writeFile(data,file);
    }
    public static void main(String []s){
        System.out.println("1");
    }
}
